### DISCUSSION TITLE
Cannot assign the left and right buttons on the scrolling wheel

### DISCUSSION THREAD

**Discussion by OnurTalha** [noah-nuebling/mac-mouse-fix#594] [Questions]
Hey everyone,
I've got an HP Envy Travel Bluetooth Mouse, and this mouse has 2 additional buttons on the scrolling wheel. I want to assign these buttons as switching between desktops. When I try to add a button on the app, it just wouldn't capture the action of me pressing the button to assign an action. 
![IMG_1991](https://github.com/noah-nuebling/mac-mouse-fix/assets/76171262/df215851-2fd5-4992-9cff-87ce77f5eea2)

**Comment by Ralphster45** [comment]
Hi,

I have a Dell MS7421W.

Exactly the same question.

otherwise, great app.

cheers

Ralph
